tests/ui_tools/test_boxes.py

```diff
@@ -26,6 +26,9 @@ def write_box(self, mocker, users_fixture, user_groups_fixture,
         write_box = WriteBox(self.view)
         write_box.view.users = users_fixture
         write_box.model.user_dict = user_dict
+        write_box.model.max_stream_name_length = 60
+        write_box.model.max_topic_length = 60
+        write_box.max_message_length = 10000
```

Comment:
This typo (?) concerns me - do we even need these values?

Reply:
These are necessary for the tests to pass since the `ReadlineEdit` gets these values from model, and if we don't define this ourselves, `Mock` will be used in their place.

Reply:
I meant specifically the last line - which seems like a typo in that you have no `model` in there. Do mock values just work anyhow?

Reply:
Ahh...Right, my bad I missed that.

I just tested with mock values for all, and it fails as expected, for the above two but not for the `max_message_length` field. On looking at the test summary, it appears to fail because the above fields pass an `edit_text` parameter to the `ReadlineEdit`, but the `msg_write_box` does not. And, passing this parameter leads to the below condition in `urwid_readline/readline_edit.py` being positive, and `max_char` being of type `Mock`, can't be used as an index in slicing:
```python3
if max_char and "edit_text" in kwargs:
    kwargs["edit_text"] = kwargs["edit_text"][:max_char]
```
So, `Mock` values should work here if we don't specify `edit_text` parameter to ReadlineEdit initially. I have updated the PR fixing the above typo.